dokka-runners/dokka-gradle-plugin/src/test/kotlin/DokkaPluginTest.kt

```diff
@@ -115,5 +120,74 @@ class DokkaPluginTest : FunSpec({
                 }
             }
         }
+
+        context("DokkaPublications") {
+
+            context("dokka dsl provides convention values for suppressInheritedMembers and suppressObviousFunctions") {
```

Comment:
nit: I think a more accurate description of this test is that "extension level suppression settings propagate to published suppressions by convention" or something like that